.vscode/launch.json

```diff
@@ -89,6 +89,32 @@
             "request": "launch",
             "url": "http://localhost:8080/",
             "webRoot": "${workspaceRoot}"
+        },
+        {
+            // http://blog.likewise.org/2017/02/debugging-a-javascript-webdriverio-project-in-vscode/
+            // http://www.wiktorzychla.com/2017/08/debugging-javascript-webdriver-io-in.html
+            "type": "node",
+            "request": "launch",
+            "port": 5859,
```

Comment:
why do you need it?

Reply:
That's the port wdio uses when you put it in debug mode, from http://webdriver.io/guide/testrunner/configurationfile.html

Reply:
I am a bit confused: why do you use it `execArgv` below if wdio uses it automatically?